### PR TITLE
Bug: Error when navigating to Dashboard after Login or Logout

### PR DESCRIPTION
**Pull request by PaulMEdwards**
results in blank page due to console error.

Please review/merge #57 first.

- [x] Login
- [x] Logout

Also corrected:
- Warning: Each child in a list should have a unique "key" prop

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -9,28 +9,37 @@ import SearchBarAutoComplete from './search-bar/SearchBarAutoComplete';
...
-        key="1"
-        style={{ marginTop: '15px', color: '#549bea' }}
-        onClick={logout}
-      >
-        Sign Out
-        <LogoutOutlined />
-      </Button>
-    </Link>
-      </>
+      <React.Fragment key="group">
```
Comment (PaulMEdwards) [inline src/components/NavBar.js:30]: This is what fixed the "Warning: Each child in a list should have a unique "key" prop" issue.

**Comment by PaulMEdwards** [comment]
[Bug: Navigating to / (Dashboard) after login/logout results in blank page due to console error](https://trello.com/c/8bw419P4/36-bug-navigating-to-dashboard-after-login-logout-results-in-blank-page-due-to-console-error)

**Comment by PaulMEdwards** [comment]
[Bug: Navigating to / (Dashboard) after login/logout results in blank page due to console error](https://trello.com/c/8bw419P4/36-bug-navigating-to-dashboard-after-login-logout-results-in-blank-page-due-to-console-error)

**Comment by PaulMEdwards** [comment]
[Bug: Navigating to / (Dashboard) after login/logout results in blank page due to console error](https://trello.com/c/8bw419P4/36-bug-navigating-to-dashboard-after-login-logout-results-in-blank-page-due-to-console-error)

**Review by Brodt7258 — APPROVED** [review]
look good, nice catch on the private route

**Review by william-jensen01 — APPROVED** [review]
Went over the changes in a call so I have no questions! Looks good

**Pull request merged**